Form-Controls/styles.css

```diff
@@ -0,0 +1,31 @@
+body {font-family: Arial, Helvetica, sans-serif;}
+* {box-sizing: border-box;}
+
+input[type=text],input[type=email] select, fieldset{
+  width: 50%;
```

Comment:
50% of what? Look at this layout in DevTools using the Device mode and have a think https://developer.chrome.com/docs/devtools/device-mode/